Dockerfile

```diff
@@ -325,6 +368,20 @@ FROM binaries AS buildkit-windows
 # this is not in binaries-windows because it is not intended for release yet, just CI
 COPY --link --from=buildkitd /usr/bin/buildkitd /buildkitd.exe
 
+# can be removed when buildkitd for windows is released
```

Comment:
@crazy-max I've just put up a PR to get the buildkitd binary for windows released: https://github.com/moby/buildkit/pull/4662/files